COLLABORATOR_GUIDE.md

```diff
@@ -116,6 +116,11 @@ oppose the PR, it can be landed. Where there is disagreement among TSC members
 or objections from one or more Collaborators, `semver-major` pull requests
 should be put on the TSC meeting agenda.
 
+#### Helpful resources
```

Comment:
I placed the links inside COLLABORATOR_GUIDE.md in a specific section since the doc is already quite long and I didn't want to clutter the text with a few links. Tell me what you think !